src/components/Templates/DefaultSlideTemplate.vue

```diff
@@ -115,7 +104,7 @@ export default {
     },
     eventBody: {
       type: String,
-      default: 'Sherbrooke event slide body. CMPT371 Team 4: Erik, Mahmoud, Jordan, Amy, Peggy, Ehtasham, Ben, Li, Ridwan, Duke, Mason, Nam'
+      default: 'Sherbrooke event slide body. Lorem ipsum dolor sit amet, pri veniam forensibus id. Vis maluisset molestiae id ad semper lobortis cum. At impetus detraxit incorrupte usu, repudiare assueverit ex eum, ne nam essent vocent admodum. '
     },
     slideModeOn: {
```

Comment:
also would we be able to remove this prop from this component, because this component should remain visible no matter what. It is the responsibility of the parent component to hide or show this component.